src/linker/Linker.Steps/LinkAttributesStep.cs

```diff
@@ -65,6 +65,10 @@ IEnumerable<CustomAttribute> ProcessAttributes (XPathNavigator nav, ICustomAttri
 				else {
 					try {
 						assembly = GetAssembly (Context, AssemblyNameReference.Parse (assemblyName));
+						if (assembly == null) {
```

Comment:
It looks like `GetAssembly` will throw whenever `assembly == null` (thus never hitting this code).

Reply:
It actually can return null (`--skip-unresolved true` will do that). There's a bug in this step but in another place - thanks for pointing this out - fixed.